packages/core/src/components/radio-checkbox/_radio-checkbox.scss

```diff
@@ -25,7 +25,7 @@
         content: '';
         background-color: $ray-color-white;
         border: $ray-radio-checkbox-border-width solid $ray-color-black;
-        width: $ray-radio-checkbox-size;
+        min-width: $ray-radio-checkbox-size;
```

Comment:
can we also set width here? to really make sure that its always a square

Reply:
fo sho